vuepress/docs/v7.0/docs/getting-started/concepts-overview.md

```diff
@@ -1,173 +1,167 @@
-
-Let’s start with a picture of an Entando cluster and how the various
-members interact with each other.
+### Architecture -Members of the Cluster
 
 ![Entando Cluster Architecture Diagram](./img/entando-architecture-v1.5.svg)
 
-### Members of the Cluster
 
-An Entando cluster is composed of various citizens which interact with
-each other. Most of these citizens have a Custom Resource Definition
-file associated with them and are deployable on Kubernetes using the
+This picture of an Entando cluster shows how the various citizens interact with each other. Most of the citizens have a Custom Resource Definition
+file associated with them. They are deployed on Kubernetes, through the
 Entando operator and controllers.
 
 #### EntandoApp
 
-An Entando application is composed of three parts:
+The EntandoApp is composed of three parts:
 
-1.  **Entando App Builder**: the user interface to customize and build an Entando application, as well as interact with the Entando Component Repository.
+1.  **Entando App Builder**: the user interface to build and customize an application.
 
-2.  **Entando App Engine**: the backend APIs providing access to Entando
+2.  **Entando App Engine**: the backend APIs providing access to 
     services.
 
-3.  **Entando Component Manager**: the service providing the Entando
-    Component Repository functionality, e.g. listing the available
-    bundles, install/uninstall a bundle, etc.
+3.  **Entando Component Manager (ECM)**: the service integrating the ECR into the App Builder providing bundle management for deploying and versioning components.
 
-The interaction between these three components (and the rest of the
-Entando cluster) use the authorization/authentication features provided
-by Keycloak.
+The interaction between these three components, and the rest of the cluster, use the authorization/authentication features provided by Keycloak.
 
 #### Entando Component Manager
 
-As briefly introduced before, the [Entando Component Manager](../compose/ecm-overview.md) is able to
-list the EntandoDeBundles accessible from the EntandoApp and provide the
-install/uninstall services to install a bundle on an Entando App. All of
-these services are made possible by the communication with the Entando
-Kubernetes service, the only service of the Entando ecosystem (other
-than the operator itself) able to interact with the cluster and some of
-the Entando Custom Resources.
+The [Entando Component Manager](../compose/ecm-overview.md) lists the EntandoDeBundles accessible from the EntandoApp and provides the install/uninstall/upgrade/downgrade services to manage a bundle. The K8s service communicates with the cluster to provide these services. It is the only service, other than the operator, that can interact with the cluster and the required custom resources. 
 
 #### Entando Kubernetes Service
 
-The Entando Kubernetes Service is part of the Entando cluster
+The Entando K8s service is part of the cluster
```

Comment:
Minor, but maybe Kubernetes integration service per the glossary. I wouldn't shorten it here in the full name like this.